server/backend-api/app/db/mongo.py

```
@@ -1,9 +1,12 @@
 import motor.motor_asyncio
+import logging
 from app.core.config import settings
 
+logger = logging.getLogger(__name__)
+
 MONGO_URI = settings.MONGO_URI
-MONGO_DB = "smart-attendance" 
+MONGO_DB = "smart-attendance"
 
-print(f"Connecting to MONGO_URI: {MONGO_URI}")
+logger.info(f"Connecting to MongoDB...")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)